COMETwebapp/SessionManagement/SessionAnchor.cs

```diff
@@ -160,10 +166,12 @@ public IEnumerable<DomainOfExpertise> GetModelDomains(EngineeringModelSetup? mod
         /// </summary>
         public async Task RefreshSession()
         {
+            CDPMessageBus.Current.SendMessage<SessionStateKind>(SessionStateKind.Refreshing);
             var actualIterationSetup = this.OpenIteration?.IterationSetup;
             var actualEngineeringModelSetup = this.GetSiteDirectory().Model.Find(m => m.IterationSetup.Contains(actualIterationSetup));
             await this.Session.Refresh();
             await this.GetIteration(actualEngineeringModelSetup, actualIterationSetup);
```

Comment:
i don;t think you need to call `await this,GetIteration`. 

The `Session.Refresh` on line 172 gets all new data from the server and updates all the objects in the cache, On line 173 you call `this.GetIteration` again calls a read on the Sesssion object which is not required. On line 172 all data was already retrieved.

I think the body of this method can be simplified to:

```
         /// <summary>
        /// Refresh the <see cref="ISession"/> object
        /// </summary>
        public async Task RefreshSession()
        {
             var sw = Stopwatch.StartNew();
            CDPMessageBus.Current.SendMessage<SessionStateKind>(SessionStateKind.Refreshing);

            await this.Session.Refresh();
            CDPMessageBus.Current.SendMessage<SessionStateKind>(SessionStateKind.UpToDate);
            Console.WriteLine($"Session refreshed in {sw.ElapsedMilliseconds} [ms]");
        }

```